packages/app/src/api.ts

```diff
@@ -362,16 +367,20 @@ export function getApps(): FirebaseApp[] {
  * @public
  */
 export async function deleteApp(app: FirebaseApp): Promise<void> {
-  let foundApp = false;
+  let cleanupProviders = false;
```

Comment:
Renamed this variable to make it better reflect why the code below is executed (on new line 383).